test/framework/build_log.py

```diff
@@ -65,7 +65,7 @@ def test_easybuilderror(self):
         self.assertErrorRegex(EasyBuildError, 'BOOM', raise_easybuilderror, 'BOOM')
         logToFile(tmplog, enable=False)
 
-        log_re = re.compile("^%s :: BOOM \(at .*:[0-9]+ in [a-z_]+\)$" % getRootLoggerName(), re.M)
+        log_re = re.compile("^%s :: BOOM( \(at .*:[0-9]+ in [a-z_]+\))?$" % getRootLoggerName(), re.M)
```

Comment:
not sure how happy I am with this, since this basically indicates that the location where errors occur are no longer available...

@JensTimmerman: maybe you've been a little bit too eager with putting the `if __debug__`s in place?

I'll try and see if this can be restored by rolling back one of the `if __debug__`s without big impact on performance...